### PR TITLE
Add Q3 2024 Hyperledger Cello Report

### PR DESCRIPTION
**Pull request by zhuyuanmao**
Add Q3 2024 Hyperledger Cello Report

### CODE REVIEW

**Review by petermetz — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Cello.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+parent: 2024
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# Project Health
+
+
+The Hyperledger Cello project remains in good health, marked by the recent update. This update includes key dependency upgrades and bug fixes.
```
Comment (denyeart) [inline project-reports/2024/2024-Q3-Hyperledger-Cello.md:13]: What do you mean by the 'recent update'? The last release looks like 1.0.0 in October 2023.

File: project-reports/2024/2024-Q3-Hyperledger-Cello.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* v1.0.0-beta1 was completed in November 2022.
+
+# Overall Activity in the Past Quarter
+
+The Cello team has focused on addressing issues in released versions, with significant progress on several key tasks:
+
+* Fixed and tested the issue related to uploading Chaincode.
+* Onboarded two interns to contribute to the Cello project.
+* Proposed a new server-based component to abstract business logic for creating Fabric networks.
+* Discussed and planned the key features for the next version.
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Cello.md:44]: Do you have a link to the roadmap for the project?

**Review by swcurran — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Cello.md (hunk 2 above)
Comment (zhuyuanmao) [reply project-reports/2024/2024-Q3-Hyperledger-Cello.md:44]: Thanks for the feedback, we haven't had the roadmap yet but started a thread to discuss this https://github.com/hyperledger/cello/issues/634

File: project-reports/2024/2024-Q3-Hyperledger-Cello.md (hunk 1 above)
Comment (zhuyuanmao) [reply project-reports/2024/2024-Q3-Hyperledger-Cello.md:13]: The recent update is more like maintenance work, such as bug fixes and dependency updates

**Review by tkuhrt — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

**Pull request merged**